### DISCUSSION TITLE
install 报错

### DISCUSSION THREAD

**Discussion by YnRen22852** [secretflow/secretflow#1406] [Q&A]
![image](https://github.com/user-attachments/assets/5ba84b89-3b52-4952-a824-736ce2e973d0)
安装secretflow出现上述错误该怎么解决

**Comment by pchyuan** [comment]
一般来说，这个问题很多时候都是因为环境及软件版本导致的，
需要您提供更多的环境，以及安装方式。
问题沟通，请移步issue区：https://github.com/secretflow/secretflow/issues
